Tests/SphericalExtractionTest/SimulationParameters.hpp

```diff
@@ -0,0 +1,63 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef SIMULATIONPARAMETERS_HPP_
+#define SIMULATIONPARAMETERS_HPP_
+
+// General includes
+#include "ChomboParameters.hpp"
+#include "GRParmParse.hpp"
+
+// Problem specific includes
+#include "SphericalExtraction.hpp"
+
+class SimulationParameters : public ChomboParameters
+{
+  public:
+    SimulationParameters(GRParmParse &pp) : ChomboParameters(pp)
+    {
+        read_params(pp);
+    }
+
+  private:
+    void read_params(GRParmParse &pp)
+    {
+        // extraction params
+        dx.fill(coarsest_dx);
+        origin.fill(coarsest_dx / 2.0);
```

Comment:
I think `dx` and `origin` should be in `ChomboParameters`! I think Katy and I have discussed this, as at some point I also wanted to inherit only from `ChomboParameters` and these were the only parameters I needed. Plus, it makes sense and doesn't break anything

Reply:
I think this is done in the apparent horizon finder PR (COSMOS-CTC-Cambridge/GRChombo#175) so I won't change this here to avoid dealing with conflicts later. All the more reason to get that finished asap :stuck_out_tongue: